src/gleam/bit_array.gleam

```diff
@@ -206,3 +206,18 @@ fn do_inspect(input: BitArray, accumulator: String) -> String {
     _ -> accumulator
   }
 }
+
+@target(javascript)
```

Comment:
This attribute is causing compilation to fail. Looks like it's not needed, and `@target` is generally discouraged, so it can be removed.

Reply:
removed.